### ISSUE TITLE
pt file is not work

### ISSUE THREAD

**Issue by Gor80hd** [kohya-ss/sd-webui-additional-networks#56]
Maybe I don't understand something, and I will be very glad to help. I trained a neural network through Dreambooth using LoRA. I have a bunch of LoRA(.pt) files with different checkpoints. In the form of ckpt they work. But when I use them in additional networks, they don't work. They do not affect the image in any way, and even more so they do not add a trained character.

**Comment by RevyaM** [comment]
Yes, prior generating using any .pt lora files it would result in this: 
![image](https://user-images.githubusercontent.com/40909979/214639992-73731112-873d-4f2b-9d0b-40ce006e6e68.png)

They also cannot show any metadata in the additional network tab. 
Large .safetensors lora models do load just fine.

**Comment by Gor80hd** [comment]
> Да, перед генерацией с использованием любых файлов .pt lora это приведет к следующему: ![образ](https://user-images.githubusercontent.com/40909979/214639992-73731112-873d-4f2b-9d0b-40ce006e6e68.png)
> 
> Они также не могут отображать какие-либо метаданные на вкладке дополнительной сети. Большие модели .safetensors lora загружаются просто отлично.

So do I need to convert or train in another way? If you then how?

**Comment by RevyaM** [comment]
> > Да, перед генерацией с использованием любых файлов .pt lora это приведет к следующему: ![образ](https://user-images.githubusercontent.com/40909979/214639992-73731112-873d-4f2b-9d0b-40ce006e6e68.png)
> > Они также не могут отображать какие-либо метаданные на вкладке дополнительной сети. Большие модели .safetensors lora загружаются просто отлично.
> 
> So do I need to convert or train in another way? If you then how?

You need to train lora using sd-scripts (also has gui from another author) from the author, the page says those are the only lora supported atm. If you cannot be bothered just use monkey patch from the extensions tab, it takes longer but it does work most of the time.
